clowdapp.yaml

```diff
@@ -70,16 +74,16 @@ objects:
         # Should be a secret ?
         - name: SECRET_KEY_BASE
           value: "secret_key_base"
-    - name: compliance-import-ssg
+    deployments:
+    - name: db-migrations
       podSpec:
         image: ${IMAGE}:${IMAGE_TAG}
         initContainers:
         - command:
             - bash
             - -c
             - |
-              while ! bundle exec rake --trace db:abort_if_pending_migrations; do sleep 1; done;
-              bundle exec rake ssg:import_rhel_supported --trace
+              bundle exec rake --trace db:debug db:migrate
```

Comment:
Out of scope for this effort: the migrations should probably run in a regular container not in initContainer, then the "sleep" workaround would not be needed.

Reply:
Unfortunately, clowder does not allow a `restartPolicy` of `OnFailure`. It's hardcoded to `Always` so the pod must not terminate.

Reply:
Okay, I missed the fact that this is a deployment and not a job.